docs/evm-sidechain/evm-cli.md

```diff
@@ -0,0 +1,120 @@
+
+## Prerequisites
+
+- Docker
+- Docker Compose
+- Node (LTS)
+
+Install the CLI with npm:
+
+```bash
+npm install -g @peersyst/xrpl-evm-sidechain-cli
+```
+
+## Bootstrap a Network
+
+1. Create a network configuration JSON file. Modify the following example to fit your needs:
+   
+   ```JSON
+   {
+    "blockchain": { // Blockchain related configuration
+        "chainId": "1440003", // ChainId of the sidechain
+        "nodes": 2, // How many nodes will exist in the network
+        "nodes": [ // Specific configuration of the nodes of the network
+            {
+                "name": "validator-0", // Node name
+                "host": "validator-0", // Node host name (if you are going to host it externally) 
+            }
+        ],
+		"extraAccounts": [ // Array of extra accounts that will have initial balance
+            {
+                "address": "0x68f727c3cd7aeB5a04acB864B770f5aa193676Bd",
+                "balance": 10
+            }
+        ]
+    },
+    "bridge": { // Bridge configuration
+        "xrplRpc": "wss://sidechain-net1.devnet.rippletest.net", // XRPL Mainchain RPC  
+        "xrplSeed": "******", // XRPL Mainchain seed with funds to create bridge account and witnesses
+        "minCreateAmount": 50, // Bridge min create amount
+        "minRewardAmount": 1, // Bridge min reward amount
+        "quorum": 1, // Bridge quorum
+        "witnesses": [{
+            "type": "aws", // Signer type (only aws supported by now)
+            "evmAddress": "0x96329A50d10a3F69311E4f4E108672926c51c474", // Witness EVM address
+            "xrpAddress": "rpSspP5yYyomcSrgsohyKMCnu5oJsTMkYP", // Witness XRP address
+            "region": "eu-west-1", // AWS KMS Region
+            "accessKey": "******", // AWS Access Key
+            "secretAccessKey": "****", // AWS Secret access Key
+            "keyId": "****" // AWS KMS Key Id
+        }]
+
+    },
+    "explorer": { // Explorer related configuration 
+        "disable": false // Disable the explorer
+    }
+   }
+   ```
+
+2. Run the creation command with the configuration JSON file.
+   
+   ```bash
+   evmcli create --configPath ./config.json --export-path ./data
+   ```
+
+3. Run these commands to start the components:
+   
+   ```bash
+   cd ./data
+   docker-compose up -d
+   ```
+
+4. (Optional) If you enabled the local network explorer in the JSON config file, you can view it at [http://localhost](http://localhost).
+
+
+## Create a Bridge
+
+1. Create a bridge configuration JSON file. Modify the following example to fit your needs:
+   
+   ```JSON
+   {
+    "lockingChain": { // Locking chain related configuration
+        "type": "xrp", // Type of the locking chain (can be either xrp or evm)
+        "url": "wss://sidechain-net1.devnet.rippletest.net", // Node url of this chain
+        "tokenCode": "JPY", // Token code if XRP means is native bridge
```

Comment:
I don't understand the comment.  Perhaps "//Token code. If XRP, it is the native bridge."